recipes/libsc/recipe.yaml

```diff
@@ -0,0 +1,61 @@
+context:
+  version: "2.8.7"
+
+package:
+  name: libsc
+  version: ${{ version }}
+
+source:
+  url: https://github.com/cburstedde/libsc/releases/download/v${{ version }}/libsc-${{ version }}.tar.gz
+  sha256: 96028B7E16917E90DD319E9F7BBB231280FF3E74ECFE2A5480B5FF94AE1428CD
+  patches:
+    - shared_library.patch
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    - if: unix
+      then:
+        - cmake -B build -DCMAKE_INSTALL_PREFIX=$PREFIX -DSC_BUILD_SHARED_LIBS=ON -DSC_ENABLE_MPI=ON -DSC_HAVE_JSON=OFF -DSC_USE_INTERNAL_JSON=OFF
+        - cmake --build build --parallel
+        - cmake --install build
```

Comment:
Perhaps there is some C++ test that it make sense to run here? It seems that they are compiled by the CMake build.